syntaxes/haskell.YAML-tmLanguage

```diff
@@ -109,20 +109,23 @@ patterns:
           - include: '#double_colon'
           - include: '#record_decl'
           - include: '#type_signature'
-      - begin: '(\b[\p{Lu}\p{Lt}][\p{Ll}_\p{Lu}\p{Lt}\p{Nd}'']*)'
+      - begin: >-
+          (?x)
+            (\b[\p{Lu}\p{Lt}][\p{Ll}_\p{Lu}\p{Lt}\p{Nd}]*) # named constructor
+           |(\(:[\p{S}\p{P}&&[^(),;\[\]`{}_"']]+\))  # prefix operator
         beginCaptures:
           '1': {name: constant.other.haskell}
-          '2': {name: keyword.operator.double-colon.haskell}
```

Comment:
I think this was a leftover from previous code (the previous version only had one capture group anyway).

Reply:
Yeah, that looks like a bug